### ISSUE TITLE
Unable to open audio

### ISSUE THREAD

**Issue by MartinDelille** [moumar/ruby-audioinfo#16]
I tried to open a file with the given example:

```ruby
AudioInfo.open("/path/to/my/file.wav") do |info|
  info.artist   # or info["artist"]
  info.title    # or info["title"]
  info.length   # playing time of the file
  info.bitrate  # average bitrate
  info.to_h     # { "artist" => "artist", "title" => "title", etc... }
end
```

Unfortunately it doesn't work:

```
Traceback (most recent call last):
	3: from /Users/martin/.dotfiles/bin/audioinspect.rb:20:in `<main>'
	2: from /usr/local/lib/ruby/gems/2.5.0/gems/ruby-audioinfo-0.5.2/lib/audioinfo.rb:50:in `open'
	1: from /usr/local/lib/ruby/gems/2.5.0/gems/ruby-audioinfo-0.5.2/lib/audioinfo.rb:50:in `new'
/usr/local/lib/ruby/gems/2.5.0/gems/ruby-audioinfo-0.5.2/lib/audioinfo.rb:200:in `initialize': undefined method `info' for WaveFile::Reader:Class (AudioInfoError)
```

I use the following ruby version:

```
$ ruby --version
ruby 2.5.0p0 (2017-12-25 revision 61468) [x86_64-darwin16]
```

**Comment by lesliev** [comment]
Mp3Info seems to work for Wav files, though its very slow:
`Mp3Info.open('music.wav').length`

**Comment by lesliev** [comment]
Fixed in AudioInfo 0.5.3 but that gem is not released. Change to:
gem 'ruby-audioinfo', require: 'audioinfo', git: 'git@github.com:moumar/ruby-audioinfo.git'

**Comment by jaredmoody** [comment]
0.5.4 has been pushed to rubygems
